handlers/main.yml

```diff
@@ -1 +1,6 @@
 ---
+
+- name: reload icinga2 daemon
```

Comment:
Adjust the handler name to our standard

Reply:
Our ansible_guide is a bit unclear about this. Did you mean to rename the handler to `$role_name reload icinga2 daemon`?